templates/view-request/main.tpl

```diff
@@ -57,35 +57,29 @@
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $oauthProblem}
                                                 <div class="alert alert-warning mb-0">There's an issue with your account setup. Please check your OAuth configuration and ensure you've allowed the necessary grants.</div>
-                                            </div>
-                                        {elseif is_array($spoofs) && count($spoofs) > 0}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif is_array($spoofs) && count($spoofs) > 0}
                                                 <div class="alert alert-warning mb-0">This request has antispoof hits and cannot be created automatically.</div>
-                                            </div>
-                                        {elseif $requestIsBlacklisted}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $requestIsBlacklisted}
                                                 <div class="alert alert-warning mb-0">This request has title blacklist hits and cannot be created automatically.</div>
-                                            </div>
-                                        {else}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $requestStatus === 'Hospital'}
+                                                <div class="alert alert-warning mb-0">This request has been sent to the Hospital. Please try creating manually, there may be hidden Anti-spoof or Title Blacklist hits</div>
+                                            {else}
                                                 {include file="view-request/createbuttons/auto.tpl" creationMode="oauth" }
-                                            </div>
-                                        {/if}
+                                            {/if}
+                                        </div>
                                     {/if}
                                     {if $canBotCreate}
                                         {if $requestEmailSent}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createBot">
```

Comment:
> Also, I'm sure the 2 extra 'createOauth' ids should be 'createBot'

You're 100% correct here, but there's something else that I've spotted here - creation mode `2` is the bot creation mode, so the if statements (all three of them - I can't catch all in this suggestion) should be checking vs 2:

```suggestion
                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 2}d-none{/if}" id="createBot">
```

Otherwise, you get silly things like this if you have the ability to use both OAuth and Bot creations, and have your default set to OAuth:

![image](https://user-images.githubusercontent.com/722710/188517640-8fc72ad0-e960-4113-bc09-87a8fb2905b8.png)

Granted, that's a bit of an obscure bug that we'll never hit in production with the current configuration - bot creation isn't allowed (but may be once we get multi-domain properly working)

Reply:
That said, if you're working on the bot create buttons too, the same cleanup for the OAuth buttons could be applied there too to keep them similarly designed?